app/tests/test_asset_upload_endpoint.py

```diff
@@ -180,10 +180,16 @@ def test_asset_upload_create_abort_multipart(self):
         self.assertS3ObjectNotExists(key)
         number_parts = 2
         file_like, checksum_multihash = self.get_file_like_object(1 * KB)
+        offset = 1 * KB // number_parts
+        md5_parts = [{
+            'part_number': i + 1, 'md5': base64_md5(file_like[i * offset:(i + 1) * offset])
+        } for i in range(number_parts)]
```

Comment:
Because this snippet comes often, it would be better to have it as a helper method
```python
def compute_md5_parts(number_parts, offset, file_like):
    return [{
        'part_number': i + 1, 'md5': base64_md5(file_like[i * offset:(i + 1) * offset])
    } for i in range(number_parts)]
```